website/blog/2019-09-07-managing-multiples-projects-with-lerna-and-yarn-workspaces.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Managing multiples projects with Lerna and Yarn Workspaces
+author: Sergio Herrera
+authorURL: https://github.com/sergiohgz
+authorImageURL: https://avatars3.githubusercontent.com/u/14012309
+authorTwitter: sergiohgz
+---
+
+Verdaccio is a project with a big ecosystem, composed by several projects, each one with its own configurations and ways to manage. This fact transforms a simple maintenance operation, like updating a common dependency, into a real hard work.
```

Comment:
Verdaccio is a project with a big ecosystem, composed by several `projects-> dependencies`

Reply:
Here I prefer projects because we don't use all of them at the same time.